docs/roadmap/README.md

```diff
@@ -26,34 +34,39 @@ For those familiar with Rust - check out our [contribution guide](https://github
 If you have problems with code or architecture understanding - reach us at any time.
 Feeling confident and want to contribute more? - Come to [work with us](https://qdrant.join.com/)!
 
-## Milestones
+## Core Milestones
 
-* :atom_symbol: Quantization support
-  * [x] Scalar quantization f32 -> u8 (4x compression)
-  * [x] Product quantization (4x, 8x, 16x, 32x, and 64x compression)
-  * [x] Binary quantization (32x compression, 40x speedup)
-  * [ ] Support for binary vectors
+* 📃 Hybrid Search and Sparse Vectors
+  * [ ] Make Sparse Vectors serving as cheap and fast as Dense Vectors
+  * [ ] Introduce Hybrid Search into Qdrant Client
+    * [ ] Dense + Sparse + Fusion in one request
+    * [ ] Customizable Re-Ranking
```

Comment:
Do we've a rough sense of what will be allow folks to tune in the re-ranking?

Reply:
i was thinking about cross-encoders for the beginning. Especially If we can ship them pytorch-free